app/components/publishers/vacancies_component.rb

```diff
@@ -46,13 +46,15 @@ def view_applicants(vacancy, card)
     return unless vacancy.enable_job_applications?
     return unless include_job_applications?
 
-    if vacancy.job_applications.any?
-      link = govuk_link_to(I18n.t("jobs.manage.view_applicants", count: vacancy.job_applications.count),
+    if vacancy.job_applications.submitted.any?
+      link = govuk_link_to(I18n.t("jobs.manage.view_applicants", count: vacancy.job_applications.submitted.count),
                            organisation_job_job_applications_path(vacancy.id),
                            class: "govuk-link--no-visited-state")
       tag.div(card.labelled_item(I18n.t("jobs.manage.applications"), link))
-    elsif vacancy.job_applications.none?
-      text = tag.span(I18n.t("jobs.manage.view_applicants", count: 0))
+    elsif vacancy.job_applications.withdrawn.any? || vacancy.job_applications.none?
```

Comment:
This seems wrong - if there are 55 submitted applications, and 1 withdrawn application, this will evaluate to true, and it will say '0 applications'.

Reply:
`if vacancy.job_applications.submitted.any?` comes first, so wouldn't it display the number of submitted applications if there are any?

Reply:
I see, OK.

Reply:
What is the difference between `elsif vacancy.job_applications.withdrawn.any? || vacancy.job_applications.none?` and `else`?

Reply:
`  enum status: { draft: 0, submitted: 1, reviewed: 2, shortlisted: 3, unsuccessful: 4, withdrawn: 5 }`

I guess nothing lol. Will replace with an else.